css/properties/align-content.json

```diff
@@ -387,12 +387,10 @@
                   "version_added": false
                 },
                 "opera": {
-                  "version_added": false,
-                  "notes": "This value is recognized, but has no effect."
```

Comment:
Unfortunately, these notes aren't structured properly since this affects the cascade, it should be listed as `partial_implementation`. Something like this breaks in versions ???-79:

```
  align-content: space-evenly;
  align-content: end;
```

Fixing that is quite the effort, however, since we don't know _when_ the the recognition began. Instead, let's work around this by putting what we do into a note, how about this:

> Before version X, this value is recognized, but has no effect.

where X is the `version_added` value.

Then we'll be able to reconstruct the right structure later, after some testing. (Or, if you're interested in doing that testing, I can suggest an alternative structure).

This would apply to all of the notes like this in the PR.